extensions/resteasy-reactive/rest-client-reactive/runtime/src/test/java/io/quarkus/rest/client/reactive/runtime/RestClientCDIDelegateBuilderTest.java

```diff
@@ -227,7 +227,7 @@ private static RestClientConfig createSampleClientConfig() {
         clientConfig.maxRedirects = Optional.of(104);
         clientConfig.followRedirects = Optional.of(true);
         clientConfig.multipart = new RestClientMultipartConfig();
-        clientConfig.multipart.maxChunkSize = Optional.of(1024);
+        clientConfig.maxChunkSize = Optional.of(1024);
```

Comment:
Should we update the value to be the default from the config?

Reply:
This test actually tests that the value loaded from an `application.properties` is reflected onto the config object